modules/cli/src/main/scala/scala/cli/commands/InstallHome.scala

```diff
@@ -0,0 +1,39 @@
+package scala.cli.commands
+
+import caseapp._
+import coursier.env.{EnvironmentUpdate, ProfileUpdater}
+
+import java.io.File
+import scala.util.Properties
+
+object InstallHome extends ScalaCommand[InstallHomeOptions] {
+  override def hidden = true
+  override def names = List(
+    List("install", "home")
+  )
+  def run(options: InstallHomeOptions, args: RemainingArgs): Unit = {
+
+    if (Properties.isWin)
+      System.err.println(s"Install home is not supported yet on windows")
+
+    // copy scala-cli to binary directory
+    val binDirPath      = scala.build.Directories.default().binRepoDir
+    val scalaCliBinPath = binDirPath / "scala-cli"
+
+    if (os.exists(scalaCliBinPath)) os.remove(scalaCliBinPath)
+    os.makeDir.all(binDirPath)
+    os.copy(os.Path(options.scalaCliBinaryPath, os.pwd), scalaCliBinPath)
```

Comment:
Should we really wipe all of `scalaCliBinPath` here? It might be a problem if we decide to put some more binaries there later on.

Maybe we should ask the user if (s)he wants to proceed if a launcher already exists in the bin dir.